apps/s_client.c

```diff
@@ -2590,7 +2638,7 @@ int s_client_main(int argc, char **argv)
             do {
                 mbuf_len = BIO_gets(fbio, mbuf, BUFSIZZ);
                 /*
-                 * According to RFC 5804 § 1.7, capability
+                 * According to RFC 5804 搂 1.7, capability
```

Comment:
Please don't change that to Chinese. Maybe write it as "section" instead?

Reply:
ok，will change it to section.